library/elliptics.h

```diff
@@ -64,7 +64,7 @@ struct dnet_io_req {
 	struct list_head		req_entry;
 
 	struct dnet_net_state		*st;
-	struct n2_message		*n2_msg;
+	struct n2_call			*n2_call_data;
```

Comment:
Do we broke ABI here? May be put this member to end of struct?

Reply:
You're right, I'll move.